src/themes/greyVest/index.js

```diff
@@ -160,7 +160,7 @@ export let GVStyle = () => (
         margin: 0;
         background: #f6f6f6;
         font-family: Lato;
-        font-size: 16px;
+        font-size: 14px;
```

Comment:
This would've been a good opportunity to switch over the rest of the font sizes to `em` multiples of this one. There really aren't that many, though, so I'll let it slide. 🙂